crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -1341,7 +1400,9 @@ mod tests {
             ]),
         };
 
-        assert_json_snapshot!(info)
+        with_settings!({ sort_maps => true, prepend_module_to_snapshot => false }, {
```

Comment:
also, please don't set `sort_maps => true` where it wasn't before.

Reply:
Fixed by https://github.com/matrix-org/matrix-rust-sdk/pull/4625/commits/cbf1fa1ee4413cf0637b6b6f871972cbae79acfb